transport_nantes/dashboard/urls.py

```diff
@@ -1,8 +1,15 @@
 from django.urls import path
-from .views import DashboardIndex, SignatureView
+from dashboard import views
 
 app_name = 'dashboard'
 urlpatterns = [
-    path('', DashboardIndex.as_view(), name='index'),
-    path('signature/', SignatureView.as_view(), name='signature'),
+    path('', views.DashboardIndex.as_view(), name='index'),
+    path('signature/', views.SignatureView.as_view(), name='signature'),
+    path('email-campaigns/', views.EmailCampaignsDashboardView.as_view(),
+         name='email_campaigns'),
+    path('email-campaigns/user/<int:pk>/',
```

Comment:
I think this should index on slug and user_id.  (Cf. other comments about why I don't understand the need for the complexity or abstraction of `EmailCampaign` objects and also think they are modeling something we explicitly don't care about.)